source/compatibility.tex

```diff
@@ -2152,11 +2152,29 @@
 \rSec2[diff.mods.to.headers]{Modifications to headers}
 
 \pnum
-For compatibility with the C standard library,
-\indextext{library!C standard}%
+For compatibility with the C standard library\indextext{library!C standard},
 the \Cpp standard library provides the C headers enumerated
 in~\ref{depr.c.headers}, but their use is deprecated in \Cpp.
 
+\pnum
+\begin{note}
+There are no \Cpp headers for the C headers
+\tcode{<stdatomic.h>}\indextext{\idxhdr{stdatomic.h}},
+\tcode{<stdnoreturn.h>}\indextext{\idxhdr{stdnoreturn.h}},
+and \tcode{<threads.h>}\indextext{\idxhdr{threads.h}},
+nor are the C headers themselves part of \Cpp.
+The headers \tcode{<ccomplex>}\indextext{\idxhdr{ccomplex}}~(\ref{ccomplex.syn})
```

Comment:
New paragraph here (between "headers in C and not in C++" and "headers in C++ that are not the same as the corresponding C header").